primitives/hp-groth16/Cargo.toml

```diff
@@ -39,7 +39,8 @@ rstest_reuse = { workspace = true }
 hex-literal = { workspace = true }
 
 [features]
-default = ["std"]
+default = ["std", "implementation"]
+implementation = []
```

Comment:
Maybe we can find a better name ? Something like "use-wasm" ?

Reply:
From this crate point of view it's not `use-wasm` but is provide or not a groth16 implementation. I don't like `implementation` too but I didn't find any better options. Some options

1. `arkworks-impl`
2. `verify-impl`
3. `verify`